library/src/test/java/com/novoda/downloadmanager/BatchStorageRootTest.java

```diff
@@ -0,0 +1,18 @@
+package com.novoda.downloadmanager;
+
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class BatchStorageRootTest {
+
+    private final StorageRoot storageRoot = () -> "path/to/the/root/downloads";
+    private final DownloadBatchId downloadBatchId = () -> "batch_1";
+
+    @Test
+    public void appendsDownloadBatchIdToStorageRoot() {
+        BatchStorageRoot batchStorageRoot = BatchStorageRoot.with(storageRoot, downloadBatchId);
+
+        assertThat(batchStorageRoot.path()).isEqualTo("path/to/the/root/downloads/batch_1");
+    }
+}
```

Comment:
needs a new line

Reply:
@zegnus  you mean new line after last `}` ?

Reply:
Fixed.